src/domains/citizen-auth-common/03_storage.tf

```
@@ -277,3 +277,25 @@ resource "azurerm_storage_queue" "profiles_to_sanitize" {
   name                 = "profiles-to-sanitize"
   storage_account_name = module.io_citizen_auth_storage.name
 }
+
+
+module "azure_storage_account" {
+  source = "github.com/pagopa/dx//infra/modules/azure_storage_account?ref=main"
+
+  environment                          = local.itn_environment
+  resource_group_name                  = data.azurerm_resource_group.data_rg_itn.name
```

Since the data is only used to get the name of the resource group and its input is actually that name, I think we can get rid of it

```suggestion
  resource_group_name                  = "${local.project_itn}-data-rg-01"
```